testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Kafka.java

```diff
@@ -30,6 +30,9 @@
 {
     private static final String CONFLUENT_VERSION = "5.5.2";
     private static final int SCHEMA_REGISTRY_PORT = 8081;
+    static final String LOGICAL_NAME_KAFKA = "kafka";
```

Comment:
just `KAFKA`? We do not use prefix like this for `COORDINATOR` or `WORKER`

Reply:
Got it. I removed the prefix `LOGICAL_NAME_`